frontends/election-manager/src/components/election_manager.tsx

```diff
@@ -61,12 +61,19 @@ export function ElectionManager(): JSX.Element {
         <Route exact path={routerPaths.root}>
           <UnconfiguredScreen />
         </Route>
-        <Route exact path={routerPaths.advanced}>
-          <AdvancedScreen />
-        </Route>
         <Route exact path={routerPaths.electionDefinition}>
           <UnconfiguredScreen />
         </Route>
+        {!areVvsg2AuthFlowsEnabled() && (
+          <Route exact path={routerPaths.advanced}>
+            <AdvancedScreen />
+          </Route>
+        )}
+        {areVvsg2AuthFlowsEnabled() && (
+          <Route exact path={routerPaths.backups}>
+            <BackupsScreen />
+          </Route>
+        )}
```

Comment:
There are bigger changes that need to be made to this no-election-definition branch, but those will come later in https://github.com/votingworks/vxsuite/issues/1979